plugins/generator-1.20.4/neoforge-1.20.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -528,7 +514,7 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 					}
 				} else if (this.isFood(itemstack)) {
 					this.usePlayerItem(sourceentity, hand, itemstack);
-					if (this.random.nextInt(3) == 0 && !net.minecraftforge.event.ForgeEventFactory.onAnimalTame(this, sourceentity)) {
+					if (this.random.nextInt(3) == 0 && !net.neoforged.neoforge.event.EventHooks.onAnimalTame(this, sourceentity)) {

```

Comment:
FQDN may not be needed since there is only one EventHooks class

Reply:
:heavy_check_mark: Fixed